recipe/meta.yaml

```diff
@@ -55,7 +54,8 @@ test:
        "libicuuc"
      ] %}
     {% for each_lib in libs %}
-    - test -f $PREFIX/lib/{{ each_lib }}.a  # [not win]
+    # https://github.com/conda-forge/icu-feedstock/issues/40
+    # - test -f $PREFIX/lib/{{ each_lib }}.a  # [not win]
```

Comment:
Did the test fail finally? If so: please re-add with `test ! -f`; otherwise, we should look into whether the tests actually run.

Reply:
Yep, failed! Will invert.

Reply:
Did run: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=768317&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=2936